### PR TITLE
Specified the property of the object.

### PR DESCRIPTION
**Pull request by adideas**
Specified the property of the object. Where does it come from and how to change it. I hope this helps others!

### CODE REVIEW

**Comment by adideas** [comment]
I am looking into your API to create a plugin. If I see difficult moments or errors, I will create a "pull request"!

File: docs/backend-api/resources/fleet/vehicle/index.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ VIN, speed, consumption and other. Vehicle object should be assigned to tracker
 * `free_insurance_policy_number` -  string. Free insurance policy number.
 * `free_insurance_valid_till` - string date. The date till free insurance valid.
 * `icon_id` - nullable int. Can only be updated via [avatar/assign](../vehicle/avatar.md#assign).
-* `avatar_file_name` - string. File name.
+* `avatar_file_name` - string. The name of the avatar file. This name is set when uploading a file via [avatar/upload](../vehicle/avatar.md#upload). Can only be updated via [avatar/assign](../vehicle/avatar.md#assign).
```
Comment (john9x) [inline docs/backend-api/resources/fleet/vehicle/index.md:96]: > Can only be updated via [avatar/assign](../vehicle/avatar.md#assign). 

`avatar/assign` does not change `avatar_file_name`

Comment (adideas) [reply docs/backend-api/resources/fleet/vehicle/index.md:96]: Ok, thank you. I thought it was logical)

**Comment by adideas** [comment]
IClosed because I made a mistake

File: docs/backend-api/resources/fleet/vehicle/index.md (hunk 1 above)
Comment (john9x) [reply docs/backend-api/resources/fleet/vehicle/index.md:96]: > I thought it was logical

You are right but it is possible to get vehicle with icon and uploaded image for now.

**Pull request closed without merging**